clients/drcachesim/tests/invariant_checker_test.cpp

```diff
@@ -461,13 +461,78 @@ check_function_markers()
     return true;
 }
 
+bool
+check_repeated_syscall_with_same_pc()
+{
+    constexpr addr_t ADDR_ONE = 0x7fcf3b9dd9e9;
+    constexpr addr_t ADDR_TWO = 0x7fcf3b9dd9eb;
+    constexpr addr_t ADDR_THREE = 0x7fcf3b9dd9ed;
+    // Negative: syscalls with the same PC.
+#if defined(X86_64) || defined(X86_32) || defined(ARM_64)
+    {
+        std::vector<memref_t> memrefs = {
+            gen_marker(1, TRACE_MARKER_TYPE_FILETYPE, OFFLINE_FILE_TYPE_ENCODINGS),
+#    if defined(X86_64) || defined(X86_32)
+            gen_instr_encoded(1 /*tid*/, 0 /*pc*/, { 0x01 }),
```

Comment:
Since we have the same value of tid for all these calls, I'd suggest we specify a default value in the definition of gen_instr_encoded so that we don't need to specify it repeatedly.

Reply:
Done.